README.md

```diff
@@ -101,12 +102,11 @@ are required)
 
 ```bash
 DOCKER_BUILDKIT=1 docker build \
-                --tag vpf-gpu-all \
-                -f docker/Dockerfile.gpu \
+                --tag vpf-gpu \
+                -f ocker/Dockerfile \
                 --build-arg GEN_PYTORCH_EXT=1 \
-                --build-arg GEN_OPENGL_EXT=1 \
                 .
-docker run -it --rm --gpus=all vpf-gpu-all
+docker run -it --rm --gpus=all vpf-gpu
```

Comment:
Can we remove `. Check the [Wiki page](https://github.com/NVIDIA/VideoProcessingFramework/wiki/Building-from-source) on how to build from source.` from README

Reply:
done